src/averageAges.js

```diff
@@ -15,11 +15,23 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with logical operators (&&, ||) or ternary operator (?:)
-  // without nesting
+  let men;
+  if (typeof century === 'undefined') {
+    men = people.filter((person) => {
+      return person.sex === 'm';
+    });
+  } else {
+    men = people.filter((person) => {
+      return (person.sex === 'm') &&
```

Comment:
unneeded parenthesis
Also, try to move if inside a filter callback. Your code will become more shorter